modal-go/sandbox.go

```diff
@@ -88,6 +88,23 @@ func newSandbox(ctx context.Context, sandboxId string) *Sandbox {
 	return sb
 }
 
+// SandboxFromId returns a running Sandbox object from an ID.
+func SandboxFromId(ctx context.Context, sandboxId string) (*Sandbox, error) {
+	ctx, err := clientContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = client.SandboxWait(ctx, pb.SandboxWaitRequest_builder{
+		SandboxId: sandboxId,
+		Timeout:   0,
+	}.Build())
+	if err != nil {
```

Comment:
Thanks, but I think this risks swallowing other, non-`NotFound` errors too? We check more granularly in some other places, like here: https://github.com/modal-labs/libmodal/blob/e2e82fbcd4510e0c8c4822482df1b4eeea4d5189/modal-go/app.go#L75-L80